app/src/main/res/layout/scorable_category_row_item.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/always_visible_linear_layout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:clickable="true"
+    android:orientation="horizontal"
+    >
+
+    <TextView
+        android:id="@+id/category_item_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textAppearance="@style/AvenirNext_DemiBold"
+        android:textColor="@color/colorAccent"
+        />
+
+    <LinearLayout
+        android:id="@+id/expandable_relative_layout"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/expandable_section_height"
+        android:layout_below="@+id/category_item_title"
+        android:clickable="true"
```

Comment:
In theory, yes. if the user is to be able to click the alwaysVisible portion to show the hidden content and vice versa click temporarily Visible hidden content to make it hidden again. Based on convo w/ design